### PR TITLE
[HAM] Add Sleep Detector, Stress Monitor documentation and guide

### PR DESCRIPTION
**Pull request by lbardeli**
ACR: TWDAPI-178

Signed-off-by: Lukasz Bardeli <l.bardeli@samsung.com>

### CODE REVIEW

File: docs/application/web/guides/sensors/ham.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ The main features of the Human Activity Monitor API include:
 
 - Recognizing activity
 
-  You can [recognize activities](#recognizing-an-activity), or [determine whether the user is sleeping](#monitoring-sleep).
+  You can [recognize activities](#recognizing-an-activity), [determine whether the user is sleeping](#detecting-sleep), monitor user [sleep](#monitoring-sleep) or [stress level](#monitoring-stress).
```
Comment (rwalczyna) [inline docs/application/web/guides/sensors/ham.md]: [...] user's [sleep] [...]

File: docs/application/web/guides/sensors/ham.md [hunk 2]
```diff
@@ -310,6 +310,92 @@ Learning how to detect whether the user is asleep is a basic Human Activity Moni
...
+
+   function listener(label)
+   {
+       console.log("Stress level: " + label);
+   }
+
+   var ranges = [new tizen.StressMonitorDataRange("Normal",10, 50),
+                 new tizen.StressMonitorDataRange("Stress Alarm 1",0, 20),
+                 new tizen.StressMonitorDataRange("Stress Alarm 2",50),
+                 new tizen.StressMonitorDataRange("Stress Alarm 3",30,100)];
```
Comment (rwalczyna) [inline docs/application/web/guides/sensors/ham.md]: Spaces after commas are inconsistent.

File: docs/application/web/guides/sensors/ham.md [hunk 3]
```diff
@@ -310,6 +310,92 @@ Learning how to detect whether the user is asleep is a basic Human Activity Moni
...
+   {
+       listenerId = tizen.humanactivitymonitor.addStressMonitorChangeListener(ranges, listener, errorCallback);
+   }
+   catch (error)
+   {
+       console.log(error.name + ": " + error.message);
+   }
+   ```
+
+2. To enable receiving data it is need to call sensor `start()` method with `HumanActivityType` set to STRESS_MONITOR:
```
Comment (rwalczyna) [inline docs/application/web/guides/sensors/ham.md]: [...] sensor's `start()` [...]

File: docs/application/web/guides/sensors/ham.md [hunk 4]
```diff
@@ -310,6 +310,92 @@ Learning how to detect whether the user is asleep is a basic Human Activity Moni
...
+   {
+       console.log(error.name + ": " + error.message);
+   }
+
+   function listener(label)
+   {
+       console.log("Stress level: " + label);
+   }
+
+   var ranges = [new tizen.StressMonitorDataRange("Normal", 10, 50),
```
Comment (pkosko) [inline docs/application/web/guides/sensors/ham.md]: Normal state is 10 to 50, but alarms are 0-20 and 50-infinity and 30-100. Please change labels or reorganize ranges.

File: docs/application/web/api/5.0/device_api/wearable/tizen/humanactivitymonitor.html [hunk 5]
```diff
@@ -2750,7 +3144,7 @@ <h4>Methods</h4>
 </div>
 </div>
 <div class="interface" id="HumanActivityReadRecorderSuccessCallback">
-<a class="backward-compatibility-anchor" name="::HumanActivityMonitor::HumanActivityReadRecorderSuccessCallback"></a><h3>2.22. HumanActivityReadRecorderSuccessCallback</h3>
+<a class="backward-compatibility-anchor" name="::HumanActivityMonitor::HumanActivityReadRecorderSuccessCallback"></a><h3>2.25. HumanActivityReadRecorderSuccessCallback</h3>
 <div class="brief">
  The HumanActivityReadRecorderSuccessCallback interface is a callback interface that is invoked when recorded human activity data is successfully read.
           </div>
```
Comment (mijin85cho) [inline docs/application/web/api/5.0/device_api/wearable/tizen/humanactivitymonitor.html]: Please change `2.23. GestureRecognitionCallback` in line 3189 into `2.26. GestureRecognitionCallback`.

Comment (lbardeli) [reply docs/application/web/api/5.0/device_api/wearable/tizen/humanactivitymonitor.html]: Done

File: docs/application/web/guides/sensors/ham.md [hunk 6]
```diff
@@ -310,6 +310,90 @@ Learning how to detect whether the user is asleep is a basic Human Activity Moni
    tizen.humanactivitymonitor.stop('SLEEP_MONITOR');
    ```
 
+## Detecting Sleep
+
+Learning how to detect whether the user is asleep is a basic Human Activity Monitor (HAM) management skill:
+
+1. To enable the monitor and start collecting data, use the `start()` method of the `HumanActivityMonitorManager` interface (in [mobile](../../../../org.tizen.web.apireference/html/device_api/mobile/tizen/humanactivitymonitor.html#HumanActivityMonitorManager) and [wearable](../../../../org.tizen.web.apireference/html/device_api/wearable/tizen/humanactivitymonitor.html#HumanActivityMonitorManager) applications) with
```
[the remark this answers is not in the document]
Comment (lbardeli) [reply docs/application/web/guides/sensors/ham.md]: Done

File: docs/application/web/guides/sensors/ham.md [hunk 7]
```diff
@@ -310,6 +310,90 @@ Learning how to detect whether the user is asleep is a basic Human Activity Moni
...
+`HumanActivityType` set to SLEEP_DETECTOR:
+   ```
+   tizen.humanactivitymonitor.stop('SLEEP_DETECTOR');
+   ```
+
+## Monitoring Stress
+
+Learning how to register a listener that allows you to monitor user's stress is a basic Human Activity Monitor (HAM) management skill:
+
+1. To register an event handler for monitoring user stress, use the `addStressMonitorChangeListener()`  method of the `HumanActivityMonitorManager` interface (in [mobile](../../../../org.tizen.web.apireference/html/device_api/mobile/tizen/humanactivitymonitor.html#HumanActivityMonitorManager) and [wearable](../../../../org.tizen.web.apireference/html/device_api/wearable/tizen/humanactivitymonitor.html#HumanActivityMonitorManager) applications):
```
[the remark this answers is not in the document]
Comment (lbardeli) [reply docs/application/web/guides/sensors/ham.md]: Done

File: docs/application/web/guides/sensors/ham.md [hunk 8]
```diff
@@ -324,7 +408,9 @@ The following table introduces the available monitor types and lists the monitor
 | Heart rate monitor                   | `http://tizen.org/feature/sensor.heart_rate_monitor` | When the heart rate monitor (HRM) sensor is started, a change callback is invoked when data changes. Use the `getHumanActivityData()` method to get the current data. |
 | GPS                                  | `http://tizen.org/feature/location.batch` | When the GPS sensor is started, a change callback is invoked when data changes. Use the `getHumanActivityData()` method to get the current data.	 The GPS sensor provides both the current value and a short history of last recorded GPS positions. The sensor supports sampling intervals, which can be used to create more power-efficient applications. |
 | Sleep monitor                        | `http://tizen.org/feature/sensor.sleep_monitor` | When the sleep sensor is started, a change callback is invoked when data changes. Use the `getHumanActivityData()` method to get the current data. |
+| Sleep detector                        | `http://tizen.org/feature/sensor.sleep_monitor` | When the sleep sensor is started, a change callback is invoked when data changes. Use the `getHumanActivityData()` method to get the current data. |
```
[the remark this answers is not in the document]
Comment (lbardeli) [reply docs/application/web/guides/sensors/ham.md]: Yes, sleep_monitor and sleep detector have the same feature

**Comment by mijin85cho** [comment]
pkosko have approved this PR, and review comments by me are all updated. So I consider this PR as been approved and merge it.

**Comment by mijin80lee** [comment]
TD Sync Completed.(for Tizen_5.0_prepare)

**Pull request merged**